### PR TITLE
DCF PROD RELEASE Q3 1596041133

### PR DESCRIPTION
**Pull request by themarcelor**
copying files from nci-crdc-staging.datacommons.io to nci-crdc.datacommons.io

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)

File: nci-crdc.datacommons.io/manifest.json [hunk 1]
```diff
@@ -35,18 +35,18 @@
...
+      "indexing": "quay.io/cdis/indexs3client:2020.07"
     }
   },
   "sower": [
     {
       "name": "manifest-indexing",
       "action": "index-object-manifest",
       "activeDeadlineSeconds": 86400,
-      "serviceAccountName": "jobs-nci-crdc-datacommons-io",
+      "serviceAccountName": "jobs-nci-crdc-staging-datacommons-io",
```
Comment (emalinowski) [inline nci-crdc.datacommons.io/manifest.json:46]: This should not be changed

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)

File: nci-crdc.datacommons.io/manifest.json [hunk 2]
```diff
@@ -84,10 +84,10 @@
       "name": "indexd-manifest",
       "action": "download-indexd-manifest",
       "activeDeadlineSeconds": 86400,
-      "serviceAccountName": "jobs-nci-crdc-datacommons-io",
+      "serviceAccountName": "jobs-nci-crdc-staging-datacommons-io",
```
Comment (emalinowski) [inline nci-crdc.datacommons.io/manifest.json:87]: This should not be changed

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)

File: nci-crdc.datacommons.io/manifest.json (hunk 1 above)
Comment (themarcelor) [reply nci-crdc.datacommons.io/manifest.json:46]: wow, nice catch 👏

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)

**Pull request merged**